__tests__/api.test.js

```diff
@@ -143,22 +143,87 @@ describe("/api/articles", () => {
 					const dateCorrected = body.articles.map((article) => {
 						article.created_at = Date.parse(article.created_at);
 						return article;
-					});
 
+					});
 					expect(dateCorrected).toBeSortedBy("created_at", {
 						descending: true,
-					});
-				});
-		});
+					})
+				})
+
+
+		})
 		it("200: should return the articles with no body property", () => {
 			return request(app)
 				.get("/api/articles")
 				.expect(200)
 				.then(({ body }) => {
 					body.articles.forEach(article => {
-					expect(article).not.toHaveProperty("body");
+						expect(article).not.toHaveProperty("body");
+					})
 				})
-				});
+		})
+
+		describe("GET /api/articles/:article_id/comments", () => {
+			it("200: should respond with an object containing an array of comment objects on the key of comments", () => {
+				return request(app)
+					.get("/api/articles/1/comments")
+					.expect(200)
+					.then(({ body }) => {
+						body.comments.forEach((comment) => {
+							expect(comment).toMatchObject({
+								comment_id: expect.any(Number),
+								votes: expect.any(Number),
+								created_at: expect.any(String),
+								author: expect.any(String),
+								body: expect.any(String),
+								article_id: 1,
+
+							});
+						});
+					});
+			});
+			it("200: should return the comments sorted by most recent first", () => {
+				return request(app)
+					.get("/api/articles/1/comments")
+					.expect(200)
+					.then(({ body }) => {
+						const dateCorrected = body.comments.map((comment) => {
+							comment.created_at = Date.parse(comment.created_at);
+							return comment;
```

Comment:
This test isn't showing that the comments are sorted by most recent first. You can just use jest sorted like you did on the previous ticket 👍